UnityProject/Assets/Scripts/Managers/SubSceneManager/SubSceneManager.SceneList.cs

```diff
@@ -90,9 +90,19 @@ IEnumerator ServerLoadMainStation(SubsceneLoadTimer loadTimer)
 		//Reset map selector
 		AdminForcedMainStation = "Random";
 
-		loadTimer.IncrementLoadBar($"Loading {serverChosenMainStation}");
-		//load main station
-		yield return StartCoroutine(LoadSubScene(serverChosenMainStation, loadTimer));
+		if(!GameManager.Instance.onTuto)
```

Comment:
can you make it so "PainStation" is chosen when you make ``onTuto`` true instead?, so this if check wouldn't be needed anymore.